### ISSUE TITLE
Cannot turn on sidebar auto expand from settings checkbox

### ISSUE THREAD

**Issue by forivall** [EnixCoda/Gitako#254]
In the developer console, when I click the checkbox in the settings, I see

```
instead of passing the 'disabled' prop directly to the input component, it should be passed to the parent component, <FormControl> 25 [content.js:73:359550](moz-extension://973cd7b7-0671-4ab9-9684-a0b8fb56c767/content.js)
```

Firefox 102.0.1
Gitako version 3.8.2

After clicking the pushpin icon in the top corner, it does turn on, and the checkbox works. But after turning it off, I can't turn it on

I presume that the control is supposed to be disabled when the sidebar isnt pinned.

**Comment by EnixCoda** [comment]
> I presume that the control is supposed to be disabled when the sidebar isnt pinned.

Exactly.

Thank you! Will fix it in the next release.

**Comment by EnixCoda** [comment]
Fixed in `v3.8.3`

**Comment by forivall** [comment]
Confirmed fixed! Thank you :heart:
